media-fonts/polarsys-b612-fonts/polarsys-b612-fonts-1.008.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit font
 
 DESCRIPTION="Font designed for aircraft cockpit displays"
 HOMEPAGE="https://b612-font.com/"
 SRC_URI="https://github.com/polarsys/b612/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
-LICENSE="OFL-1.1"
+LICENSE="EPL-1.0 BSD OFL-1.1"   # to be clarified #746725
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
 IUSE=""
```

Comment:
`IUSE` is redundant and could be dropped. Otherwise LGTM.

@luke-jr: I'm aware that this may not be the last word about the license, but we need some interim solution. In any case, the "and" leans on the safe side.

Reply:
If there was any real question, I would agree - but the intent is unambiguously clear IMO.